### ISSUE TITLE
rzmq compat breaks ffi-rzmq

### ISSUE THREAD

**Issue by gamemachine** [andrewvc/em-zeromq#16]
I see it looks like it's fixed in master.  Can we get a new release cut soon?  em-zeromq completely breaks the ability to use ffi-rzmq directly, without serious monkey patching, because of how it changes what recv returns.

Chris

**Comment by schmurfy** [comment]
The compat module was just a temporary measure and is now removed in master, I am currently away but I can make a release at the end of the week when I get back (I was just checking my mails), until then you can use the git repository easily if you use bundler by doing this:

```
gem "em-zeromq", :git => "git@github.com:andrewvc/em-zeromq.git"
```

**Comment by schmurfy** [comment]
I released 0.3.0, the monkey patch is gone.
